src/status_im/ethereum/stateofus.cljs

```diff
@@ -4,8 +4,16 @@
 
 (def domain "stateofus.eth")
 
-(defn subdomain [username]
-  (str username "." domain))
+(defn subdomain
```

Comment:
i would do it outside this function, we shouldn't ask for subdomain for stateofus names